docs/pages/docs/features/pipelines.mdx

```diff
@@ -129,21 +133,49 @@ A common pattern in many TypeScript monorepos is to declare that a package's `bu
 
 An empty dependency list (`dependsOn` is either undefined or `[]`) means that a task can start executing whenever it can! After all, it has NO dependencies.
 
-```json
+```jsonc
 {
   "$schema": "https://turborepo.org/schema.json",
   "pipeline": {
     // ... omitted for brevity
 
-    // A package's `lint` command has no dependencies and can be run at
-    // whenever.
     "lint": {
+      // A package's `lint` command has no dependencies and can be run at
+      // whenever.
+      "inputs": ["**/*.js", "**/*.ts"],
```

Comment:
```suggestion
      "inputs": ["src/**/*.js", "src/**/*.ts"],
```

Reply:
Removed this one, I switched to using `test` as an example rather than `lint`. `lint` is currently used to demonstrate not having dependencies.